index.html

```diff
@@ -70,6 +70,163 @@ <h1>
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+        <div class="project-card">
+          <div class="project-card_image">
+            
+          </div>
+          <div class="project-card_details">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+        <div class="project-card">
+          <div class="project-card_image">
+            
+          </div>
+          <div class="project-card_details">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <div class="about-me">
+      <h2>About Me</h2>
+      <img src="./images/Mask Group.png" class="background-top-right" alt="">
+      <img src="./images/Frame.svg" class="background-bottom-left" alt="">
+      <p class="content-paragraph"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      <button class="green-button about-button">Get my resume</button>
+      <div class="skills">
+        <ul class="skills-list">
+          <li>
+            <div class="skill-list">
+              <img src="./images/icon-languages.svg" alt="">
+              <h3>Languages</h3>
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+          </li>
+          <li>
+            <div class="skill-list">
+              <img src="./images/icon-frameworks.svg" alt="">
+              <h3>Frameworks</h3>
+              <ul>
+                <li>React.js</li>
+                <li>Ruby on Rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li>Selenium</li>
+              </ul>
+            </div>
+          </li>
+          <li>
+            <div class="skill-list">
+              <img src="./images/icon-skills.svg" alt="">
```

Comment:
- [ ] This section looks great but your images are missing an `alt attribute`. :thinking: Currently you left them empty, following the best practices, all images are meant to have an alt tag. **The purpose of alt text is to improve accessibility by describing what an image is showing to visitors who do not have the ability to see them. However, it also helps search engine crawlers and so improves SEO.** :smile: :+1:

Reply:
Sorry I forgot that but now it''s fixed :wrench: :+1: